src/shared/components/Studio/ExpandableStudioList.less

```diff
@@ -0,0 +1,38 @@
+@import '../../lib.less';
+
+.expandable-studio-list {
+  .more-button {
+    border: none;
+    padding: 0;
+    color: @link-color;
+    outline-style: none;
+    cursor: pointer;
+    background-color: inherit;
+  }
+
+  .studio-title-panel {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .ant-collapse-arrow {
+    top: 0 !important;
```

Comment:
Can we find a way to not use !important ? We should use !important as a last resort.